Unity-Technologies/ScriptableRenderPipeline
# Hdrp decals htile fix

### Purpose of this PR
Fixes a race condition in decal shaders

---
### Release Notes
Fixes occasional graphics artifacts in decal rendering

---
### Testing status
https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=hdrp-decals-htile-fix&unity_branch=trunk&automation-tools_branch=master

**Manual Tests**: What did you do?
- [ check] Opened test project + Run graphic tests locally
- [ check] Built a player
- [ check] C# and shader warnings (supress shader cache to see them)
- Other: 

**Automated Tests**: What did you setup?

Any test projects to go with this to help reviewers?

---
### Overall Product Risks
**Technical Risk**: Low

**Halo Effect**: Low

---
### Comments to reviewers
Notes for the reviewers you have assigned.
